solidus_starter_frontend.gemspec

```diff
@@ -33,4 +33,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop-rails'
   s.add_development_dependency 'rubocop-rspec'
   s.add_development_dependency 'rubocop-i18n'
+  s.add_development_dependency 'selenium-webdriver'
```

Comment:
Shouldn't this be already added by `solidus_dev_support`?

Reply:
I'm checking, but it doesn't seem to be included there.